Examples/Io/Performance/ActsExamples/Io/Performance/SeedingPerformanceWriter.cpp

```diff
@@ -27,23 +27,20 @@ ActsExamples::SeedingPerformanceWriter::SeedingPerformanceWriter(
     : WriterT(cfg.inputSeeds, "SeedingPerformanceWriter", lvl),
       m_cfg(std::move(cfg)),
       m_effPlotTool(m_cfg.effPlotToolConfig, lvl) {
-  if (m_cfg.inputSeeds.empty()) {
-    throw std::invalid_argument("Missing input seeds collection");
+  if (m_cfg.inputMeasurementParticlesMap.empty()) {
+    throw std::invalid_argument("Missing hit-particles map input collection");
   }
   if (m_cfg.inputParticles.empty()) {
     throw std::invalid_argument("Missing input particles collection");
   }
   if (m_cfg.outputFilename.empty()) {
     throw std::invalid_argument("Missing output filename");
   }
-  if (m_cfg.inputMeasurementParticlesMap.empty()) {
-    throw std::invalid_argument("Missing hit-particles map input collection");
-  }
 
   // the output file can not be given externally since TFile accesses to the
   // same file from multiple threads are unsafe.
   // must always be opened internally
-  auto path = m_cfg.outputFilename;
+  auto path = joinPaths(cfg.outputDir, cfg.outputFilename);
```

Comment:
Thanks a lot @msmk0 for updating the example!
Maybe this is m_cfg instead of cfg?

Reply:
Yes, I will fix that.

Reply:
Should be fixed.